dbms/src/Storages/DeltaMerge/ColumnFile/ColumnFileInMemory.h

```diff
@@ -55,7 +55,10 @@ class ColumnFileInMemory : public ColumnFile
     explicit ColumnFileInMemory(const ColumnFileSchemaPtr & schema_, const CachePtr & cache_ = nullptr)
         : schema(schema_)
         , cache(cache_ ? cache_ : std::make_shared<Cache>(schema_->getSchema()))
-    {}
+    {
+        rows = cache->block.rows();
+        bytes = cache->block.bytes();
+    }
```

Comment:
why this only affect disagg mode?

Reply:
Because in non disagg mode we create empty CFInMemory and use append() to write data.

In disagg mode we directly create a CFInMemory with Cache data.

Reply:
get, thx